src/python/WMCore/WebTools/Root.py

```diff
@@ -92,10 +91,10 @@ def access(self):
             # request.rfile.rfile.bytes_read is a custom CMS web
             #  cherrypy patch not always available, hence the test
             rbytes = (getattr(request.rfile, 'rfile', None)
-                 and getattr(request.rfile.rfile, "bytes_read", None)
-                 and request.rfile.rfile.bytes_read) or "-"
+                      and getattr(request.rfile.rfile, "bytes_read", None)
```

Comment:
8 spaces here instead of 4 ?

Reply:
that's what PyCharm decided to put in place, I guess to keep it aligned with the whole statement.